tasks/install_targz.yml

```diff
@@ -50,15 +52,41 @@
     src: app@.service.j2
     dest: "{{ instance_systemd_path }}{{ instance_systemd_unit_file }}"
   any_errors_fatal: true
-
-- name: Copy an unit-file to /etc/systemd/system/
-  become: true
-  copy:
-    remote_src: true
-    src: "{{ instance_systemd_path }}{{ instance_systemd_unit_file }}"
-    dest: "/etc/systemd/system/{{ instance_systemd_unit_file }}"
-    owner: root
-    group: root
-  any_errors_fatal: true
   notify:
     - reload-systemd-daemon
+
+- name: Configure stateboard
+  block:
+    - name: Set stateboard facts
+      set_fact:
+        stateboard_name: '{{ cartridge_app_name }}-stateboard'
+        stateboard_work_dir: '{{ cartridge_app_name | get_instance_work_dir(inventory_hostname, True, data_dir) }}'
+        stateboard_run_dir: '{{ cartridge_app_name | get_instance_run_dir(inventory_hostname, True, run_dir) }}'
+        stateboard_conf_file: '{{ cartridge_app_name | get_instance_conf_file(inventory_hostname, True, conf_dir) }}'
+
+    - name: Create stateboard folders
+      file:
+        path: "{{ item.path }}"
+        state: directory
+        mode: "{{ item.mode | default('0755') }}"
+        owner: "{{ instance_user }}"
+        group: "{{ instance_group }}"
+      loop:
+        - {path: "{{ stateboard_work_dir }}"}
+        - {path: "{{ stateboard_run_dir }}"}
+      any_errors_fatal: true
+
+    - name: Create a stateboard service file
```

Comment:
I think that we can always create this file, just like RPM and DEB packages do.
For example, you unpacked TGZ and created a cluster without specifying stateboard instance and then you want to add stateboard instance without reinstalling TGZ. Since you didn't specified that you need stateboard on first TGZ installing, you don't have stateboard unit file.